components/popover/index.css

```diff
@@ -14,94 +14,106 @@ governing permissions and limitations under the License.
 @import "@spectrum-css/commons/overlay.css";
 
 .spectrum-Popover {
-  /* Used in positioning popover for storybook */
-  --flow-direction: 1;
+	/* Used in positioning popover for storybook */
+	--flow-direction: 1;
 
-  &:dir(rtl) {
-    --flow-direction: -1;
-  }
+	&:dir(rtl) {
+		--flow-direction: -1;
+	}
 
-  /* animation distance is equal to and responsible for popover offset */
-  --spectrum-popover-animation-distance: var(--spectrum-spacing-100);
+	/* animation distance is equal to and responsible for popover offset */
+	--spectrum-popover-animation-distance: var(--spectrum-spacing-100);
 
-  --spectrum-popover-background-color: var(--spectrum-background-layer-2-color);
-  --spectrum-popover-border-color: var(--spectrum-gray-400);
+	--spectrum-popover-background-color: var(--spectrum-background-layer-2-color);
+	--spectrum-popover-border-thickness: var(--spectrum-border-width-100);
```

Comment:
It looks like there are still several references to `--spectrum-popover-border-width` which is defined in the `themes/spectrum.css`, this custom prop either needs to be updated to the same name or the references to the old one maybe should be removed and replaced... 🤔 I'm honestly not sure what the best move is here, it might be worth looking over the [Foundations PR](https://github.com/adobe/spectrum-css/pull/2786) to get a feel for how we anticipate the different themes will work together in `main`.